kolibri/plugins/user/assets/src/views/FacilitySelect.vue

```diff
@@ -106,10 +106,14 @@
     },
     methods: {
       setFacility(facilityId) {
+        let whereToNext = { ...this.whereToNext };
```

Comment:
You can make this a const, since it's not reassigned as far as i can tell
```suggestion
        const whereToNext = { ...this.whereToNext };
```